cdap-api/src/main/java/io/cdap/cdap/api/plugin/PluginConfig.java

```diff
@@ -60,6 +62,15 @@ public final PluginProperties getProperties() {
     return properties;
   }
 
+  /**
+   * Returns the raw {@link PluginProperties}
+   *
+   * No macro evaluations will be performed here, meaning that for example "schema" could map to "${schema}".
```

Comment:
Contains the properties as they were before macro evaluation. For example, at configure time, if the 'schema' property is a macro '${schema}', the value will be the raw string '${schema}' instead of null. This is primarily useful when one plugin is passing macro enabled properties to another plugin.